pkg/scaling/scale_handler.go

```diff
@@ -41,6 +41,12 @@ type scaleHandler struct {
 	recorder          record.EventRecorder
 }
 
+type scalerMetrics struct {
```

Comment:
I'd move this structure to pkg/scaling/scaledjob as well.

Reply:
Done